### PR TITLE
CXP-1104: Clean Connection Settings services

### PR DESCRIPTION
**Pull request by willy-ahva**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/Settings/controllers.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+    Akeneo\Connectivity\Connection\Infrastructure\Settings\Controller\Internal\IsConnectionsNumberLimitReachedAction:
+        public: true
+        arguments:
+            - '@Akeneo\Connectivity\Connection\Infrastructure\Settings\Persistence\IsConnectionsNumberLimitReachedQuery'
+
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/Settings/controllers.yml:6]: rm blank line

**Review by masacc — APPROVED** [review]

**Comment by guillaumefer-akeneo** [comment]
src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing/marketplace.yml to remove ? It is empty

**Comment by guillaumefer-akeneo** [comment]
src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing/marketplace.yml to remove ? It is empty

**Review by guillaumefer-akeneo — APPROVED** [review]
src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing/marketplace.yml to remove ? It is empty

**Review by RiffVentura — APPROVED** [review]

**Pull request merged**